spec/stat_calculator_spec.rb

```diff
@@ -0,0 +1,10 @@
+# spec/stat_calculator_spec.rb
+require 'rspec'
+require_relative '../lib/stat_calculator'
```

Comment:
no need to make individual requires here. We have a spec_helper file that you can add to:
just throw "require './lib/stat_calculator'" in the spec_helper file
then put "require_relative 'spec_helper'" in this file

Note for the future: require statements are for general use as well as all files that are not in the immediate directory
require_relative is only every to be used when requiring a file that is within the same directory

Reply:
Thanks for the feedback however it was added as pre-cursors for our next step. However, I installed rspec on the branch for tomorrow session with add method to the stat_calculator. I will keep the require 'rspec'. 

**Changes made based on your review:**

- Added require './lib/stat_calculator' to the spec_helper file
- Added the 'require_relative spec_helper' in the spec/stat_calculator_spec.rb file